kokoro/build_binaries.sh

```diff
@@ -28,12 +28,12 @@ chmod +x builder
 ./builder build \
  --build-config-path buildconfigs/oak_restricted_kernel_simple_io_bin.toml \
  --builder-image "$DOCKER_IMAGE_REPO_DIGEST" \
- --git-commit-digest "sha1:$COMMIT_HASH" \
+ --git-commit-digest "sha1:$KOKORO_GITHUB_PULL_REQUEST_COMMIT" \
```

Comment:
I don't think this var exist either, or did you see it anywhere? I only see `KOKORO_GITHUB_PULL_REQUEST_COMMIT_oak` in the logs

Reply:
I am getting a formatting error on it: "^--------------------^ SC2154 (warning): KOKORO_GIT_COMMIT_oak is referenced but not assigned."

Is it because of the name patter? We don't get the same error for `KOKORO_ARTIFACTS_DIR`.

I think this script should be directly in Google3.

Reply:
Yes https://www.shellcheck.net/wiki/SC2154 says that this only applies to vars with lowercase characters, that's why it didn't trigger for the other one, even though it's the same.

I don't think this script can live in google3 easily, there will be no way to keep it in sync with the repo, and the repo stuff (e.g. build configs, binary names) is more likely to change than the google3 stuff (kokoro env variables, etc). Also there would be no way of testing changes to this script as part of a PR, for instance. So I think the current location is correct.

Reply:
I think this suggestion applies here:

> If you know for a fact that the variable is set, you can use ${var:?} to fail if the variable is unset (or empty),

that's what we want after all